ts/components/helpers/withContextualHelp.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Adds a contextual help (components/ContextualHelp)
+ * to a component
+ */
+import { View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { ContextualHelp } from "../ContextualHelp";
+
+type State = {
+  isHelpVisible: boolean;
+};
+
+export type ContextualHelpInjectedProps = {
+  showHelp: () => void;
+};
+
+const styles = StyleSheet.create({
+  fullSize: {
+    width: "100%",
+    height: "100%"
+  }
+});
+
+/**
+ * Adds the methods for showing/hiding the contextual
```

Comment:
It looks like this became a shortcut for any toggable (modal) content, so maybe we can name this withModal / withTogglableContent to make it "generic" (or is there something specific to an "help" component ?)

Reply:
I would consider this a matter of definitions: my understanding of a contextual help is a "something" that offers information about the current context (e.g. the current page you're visiting) -- and this is exactly what this component does. A modal, on the other hand, offers a wider range of possibilities (various pickers, buttons, ... ), so I believe it would be a bit of a stretch to consider this a modal.